docs/source/conf.py

```diff
@@ -287,6 +283,24 @@ def process_modin_accessors(args):
             return list(map(process_modin_accessors, items))
         return items
 
+def process_signature(app, what, name, obj, options, signature, return_annotation):
+
+    # Names to remove from signature (AST related):
+    names_to_remove = ['_emit_ast', '_ast']
+
+    def remove_from_signature(signature, name_to_remove):
+        parts = [p for p in signature.split(',') if name_to_remove not in p]
+        signature = ','.join(parts)
```

Comment:
I think the dup `))` is caused by single parameter signature 

for such case `def some_func(_emit_ast:bool=True)`, `signature` will be empty string on this line, and then on line 296, a single `)` is added

Reply:
good callout, let me take a look at this

Reply:
fixed by changing logic.